app/templates/events/view/export.hbs

```diff
@@ -1,6 +1,6 @@
 <div class="ui stackable very relaxed container grid">
   <div class="eight wide column">
-    {{events/view/export/download-zip}}
+    {{events/view/export/download-zip data=model startGeneration=(action 'startGeneration') eventExportStatus=eventExportStatus eventDownloadUrl=eventDownloadUrl}}
```

Comment:
But you are passing in an action as a variable here ... `startGeneration`. So you will be able to refer to it within the `events/view/export/download-zip` component as `{{action startGeneration}}` directly just like 

- https://github.com/fossasia/open-event-frontend/blob/fbd8ddf1e5c579feb85441aadc5f63b4e7c5ee93/app/templates/components/widgets/forms/ticket-input.hbs#L27
- https://github.com/fossasia/open-event-frontend/blob/4676a0406e6dfda3bc79c656ae60cf0b7ba2854c/app/templates/components/event-card.hbs#L32